src/open_inwoner/components/templates/components/Typography/Link.html

```diff
@@ -1,7 +1,7 @@
 {% load i18n helpers icon_tags %}
 
 <a
-    href="{{href}}"
+    href="{{href}}{% if next %}?next={{next}}{% endif %}"
```

Comment:
Simply appending `?next=` breaks if the `href` already has a query string. It'd be better if a filter or template tag could do this.